Sources/VGSCollectSDK/Core/API/HTTPMethod.swift

```diff
@@ -11,6 +11,9 @@ import Foundation
 /// Key-value data type, usually used for response format.
 public typealias JsonData = [String: Any]
 
+/// Array of JsonData objects type.
+public typealias JSONArray = [JsonData]
```

Comment:
put `JsonData` in quotation marks, so in reference docs it could navigate to JsonData description

Reply:
Will remove this. We don't need `JSONArray` in public interface.

Reply:
Removed.